src/config.h.in

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#define VERSION_MAJOR @Sparselizard_VERSION_MAJOR@
+#define VERSION_MINOR @Sparselizard_VERSION_MINOR@
+#define HAS_GMSH @HAS_GMSH@
```

Comment:
I assume this config file should be used by applications to get the sparselizard version. I suggest to prefix those defines with "SPARSELIZARD_" to avoid name-clashes with other includes.

Reply:
No, these are for sparselizard itself. Unless somebody imports them.

Reply:
Added prefix